app/pages/Home.tsx

```diff
@@ -44,53 +43,33 @@ export default function Home() {
             <div className="mt-4 w-full sm:mt-10 sm:w-1/2 md:mt-10 md:w-1/2">
               <button
                 data-testid="connectWalletButton"
-                className="rounded-sm rounded bg-purple-connectPurple py-2 px-10 text-white"
+                className="invisible rounded-sm rounded bg-purple-connectPurple px-10 py-2 text-white md:visible"
                 onClick={handleConnection}
               >
                 <p className="text-base">{address ? `Disconnect from ${walletLabel || ""}` : "Connect Wallet"}</p>
               </button>
-              {address ? <div className="pt-3">Connected to: {JSON.stringify(address, null, 2)}</div> : null}
             </div>
           </div>
           {/* Stamp Provider Tile Display */}
-          <div className="container mx-auto mt-2 px-5 py-2 text-gray-600 sm:mt-20 sm:w-1/2 sm:py-20 md:mt-16 md:py-24">
+          <div className="mx-auto mt-20 bg-mobileLandingPageBackground text-gray-600 sm:mt-20 sm:w-1/2 sm:py-20 md:mt-16 md:bg-none md:py-24">
```

Comment:
there might be too much whitespace between the bottom of the text paragraph, and the logo tiles?

![Figma mock left, Passport right](https://user-images.githubusercontent.com/62616399/177361458-68020d03-6aac-4798-a29d-d30a1860d814.png)

Reply:
Resolved with https://github.com/gitcoinco/passport/pull/292/commits/c0e5d91b57f75d3f32ee4e9b945d30c2fc7fa5c2